lib/parseTriggers.js

```diff
@@ -4,15 +4,18 @@ var FirebaseError = require('./error');
 var fork = require('child_process').fork;
 var path = require('path');
 var RSVP = require('rsvp');
+var _ = require('lodash');
```

Comment:
`var isEmpty = require(lodash/isEmpty)` is much more lightweight. Requiring whole library for one method seems to be an overkill 😉

Reply:
You are correct, but Lodash is used heavily throughout the CLI and we can try to do a major refactor to clean all that up at a later date :)

Reply:
I’ve made an issue to keep track on this: https://github.com/firebase/firebase-tools/issues/635

Reply:
Thanks!